pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -993,19 +993,26 @@ func (dm *DockerManager) getSecurityOpt(pod *api.Pod, ctrName string) ([]string,
 		return nil, nil
 	}
 
+	// in Docker 1.10 the operator is :, for anything later it's =
+	optFmt := "%s=%s"
+	if result <= 0 {
+		optFmt = "%s:%s"
+	}
+	defaultSecurityOpts := []string{fmt.Sprintf(optFmt, "seccomp", defaultSeccompProfile)}
+
 	profile, profileOK := pod.ObjectMeta.Annotations[api.SeccompContainerAnnotationKeyPrefix+ctrName]
 	if !profileOK {
 		// try the pod profile
 		profile, profileOK = pod.ObjectMeta.Annotations[api.SeccompPodAnnotationKey]
 		if !profileOK {
 			// return early the default
-			return defaultSecurityOpt, nil
+			return defaultSecurityOpts, nil
 		}
 	}
 
 	if profile == "unconfined" {
 		// return early the default
-		return defaultSecurityOpt, nil
+		return defaultSecurityOpts, nil
 	}
 
 	if profile == "docker/default" {
```

Comment:
Docker default is not unconfined, but something which fits to the selected capabilities. Kubernetes chose another default than Docker here.